services/github/github-size.service.js

```diff
@@ -41,16 +60,20 @@ export default class GithubSize extends GithubAuthV3Service {
     }
   }
 
-  async fetch({ user, repo, path }) {
+  async fetch({ user, repo, path, branch }) {
     return this._requestJson({
-      url: `/repos/${user}/${repo}/contents/${path}`,
+      url: `/repos/${user}/${repo}/contents/${path}?ref=${branch}`,
       schema,
-      errorMessages: errorMessagesFor('repo or file not found'),
+      errorMessages: errorMessagesFor('repo, branch or file not found'),
     })
   }
 
-  async handle({ user, repo, path }) {
-    const body = await this.fetch({ user, repo, path })
+  async handle({ user, repo, path }, queryParams) {
+    let branch = queryParams.branch
+    if (!branch) {
+      branch = 'master'
+    }
```

Comment:
The default branch for a repo is configurable and most repos now use `main`. Instead of setting `branch` explicitly here if it is not supplied, pass a `?ref=` param in our API call only if `branch` is supplied. If the user doesn't supply the param we can leave github to work out the default.

Reply:
Hi @chris48s,
Good point, updated, thanks.